aptos-move/aptos-aggregator/src/tests/types.rs

```diff
@@ -17,11 +17,21 @@ pub fn aggregator_v1_state_key_for_test(key: u128) -> StateKey {
     StateKey::raw(key.to_le_bytes().to_vec())
 }
 
-#[derive(Default)]
 pub struct FakeAggregatorView {
     // TODO: consider adding deltas to test different read modes.
     v1_store: HashMap<StateKey, StateValue>,
     v2_store: HashMap<AggregatorID, u128>,
+    counter: RefCell<u32>,
```

Comment:
Why RefCell<u32> instead of u32?

Reply:
RefCell allows interior mutability, while otherwise you would need &mut